api/camino_common_args_responses.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2022-2024, Chain4Travel AG. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package api
+
+import "github.com/ava-labs/avalanchego/utils/formatting"
+
+// JSONEncoding contains encoding type
+type JSONEncoding struct {
```

Comment:
This arg is addded cause of change in `vms/platformvm/camino_service.go` `GetLastAcceptedBlock`
https://github.com/chain4travel/caminogo/pull/313/commits/0580b7653879e34accaceecd603cf8525d47de51#diff-817ff85c83a19e5bbb3e3469fc346d989d56b46e28fbc623174269eb1ce9f880R809
Arg itself looks pretty general, and could be used by other api handlers or be embedded into other arg/reply.

Reply:
few questions
- why is called JSONEncoding if it's encoding in general?
- service methods usually come with their own custom Args Struct. how come is this different?
- up to now this GetLastAcceptedBlock didn't actually offer a way to define encoding it was hard-coded. So maybe it was intentional to omit any specific arg structs

Reply:
1) JSONEncoding - its just a naming pattern for other avax common args, starts with `JSON`
2) Not all of them, some of them are using common structs defined in this pkg
3) It didn't do it, cause I implemented this method as a small helper for debugging, nothing more

Reply:
renamed to just `Encoding` to avoid confusion